hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDFSShell.java

```diff
@@ -576,26 +577,33 @@ public void run() {
     //use SecurityManager to pause the copying of f1 and begin copying f2
     SecurityManager sm = System.getSecurityManager();
```

Comment:
Thanks to involve me here. As #7567 mentioned, "System.getSecurityManager() will also be one Deprecate APIs based on JEP-411, Would you mind to process it together." Thanks. Others make sense to me.

Reply:
Thank you, I've moved getSecurityManager inside the try block.